packages/frontend/src/pages/Tile/index.tsx

```diff
@@ -1,10 +1,51 @@
+import { ITableMetadata, ITableRow } from '@plumber/types'
+
 import { useParams } from 'react-router-dom'
-import { Flex, Text } from '@chakra-ui/react'
+import { useQuery } from '@apollo/client'
+import { Center, Flex, Spinner, Text } from '@chakra-ui/react'
 import Container from 'components/Container'
+import { GET_ALL_ROWS } from 'graphql/queries/get-all-rows'
+import { GET_TABLE } from 'graphql/queries/get-table'
+
+import Table from './components/Table'
 
 export default function Tile(): JSX.Element {
   const { tableId } = useParams<{ tableId: string }>()
 
+  const { data: getTableData, loading: getTableLoading } = useQuery<{
+    getTable: ITableMetadata
+  }>(GET_TABLE, {
+    variables: {
+      tableId,
+    },
+  })
+
+  const { data: getAllRowsData, loading: getAllRowsLoading } = useQuery<{
+    getAllRows: ITableRow[]
+  }>(GET_ALL_ROWS, {
+    variables: {
+      tableId,
+    },
+  })
+
+  if (
+    !getTableData?.getTable ||
```

Comment:
nit: I try to sell koyok to you all on friday about convering to useSuspenseQuery and useBackgroundQuery :P

Reply:
useSuspenseQuery does look cleaner